Processing/BRCA_Deep_Dive.Rmd

```diff
@@ -3010,14 +3064,144 @@ nrow = 2
 )
 
 # combo - 
-ggplot(air_temp_data) + 
-  geom_line(aes(x = date, y = Well, color = "Groundwater"), alpha = 0.2) + 
+a2 <- ggplot(air_temp_data) + 
+  geom_line(aes(x = date, y = Well, color = "Groundwater"), alpha = 0.8) + 
+  geom_line(aes(x = date, y = Station, color = "Air"), alpha = 0.8) +
+  geom_line(aes(x = date, y = frollmean(Well, 30, na.rm = TRUE)), color = "black", lwd = 1.5) + 
   geom_line(aes(x = date, y = frollmean(Well, 30, na.rm = TRUE), color = "Groundwater"), lwd = 1) + 
-  geom_line(aes(x = date, y = Station, color = "Air"), alpha = 0.2) +
+  geom_line(aes(x = date, y = frollmean(Station, 30, na.rm = TRUE)), color = "black", lwd = 1.5) +  
   geom_line(aes(x = date, y = frollmean(Station, 30, na.rm = TRUE), color = "Air"), lwd = 1) +  
   theme_bw() + 
-  scale_color_manual("", values = c("#E70870","#002EA3")) + 
+  scale_color_manual("", values = c("#c07da5","#3070ad")) + 
   ylab("Temperature (F)") +
   xlab("Date")
 
+ggsave(a2, filename = "docs/BRCA_Deep_Dive/Figs/Figure_A2.jpg", width = 7.5, height = 3, dpi = 600)
+
+```
+
+Figure A3
+
+Frequency of extreme (Q99) precip events:
+```{r}
+df <- bind_rows(source_fut_wbm, source_hist_wbm) %>%
+  dplyr::rename(cf = CF) %>%
+  dplyr::group_by(cf) %>%
+  dplyr::mutate(rolling_3day = frollapply(precip_in_wbm, FUN = "sum", n = 3, align = "center")) %>%
+  dplyr::filter(!is.na(rolling_3day))
+
+var = "rolling_3day"
+q1 = 0.01 
+q2 = 0.99
+
+historic <- df %>%
```

Comment:
The historical calc should only include the "historical" stat years so 1980-2010.